.wakemanifest

```diff
@@ -0,0 +1,69 @@
+src/types/types.wake
```

Comment:
For testing, is this relying on the .wakemanifest being present before the tarball CI step, but not afterwards?

Reply:
The tarball case is an interesting thing to consider, but not fully sure what you are asking here. If the manifest doesn't exist then wake should fallback to the old method.

Honestly, I'm tempted to have wake still use the old method since it doesn't have the speed problem and let other repos load test the manifest method

Reply:
Yeah the wake repo using the search mechanism seems reasonable. I was asking if you were using the unusal structure of the CI to test this feature as on/off, but it could be explicitly tested instead

Reply:
no, I wasn't doing any intentional testing here. To do a good test I think we'd need some secret flag for dumping all discovered wake files

Reply:
Might be able to do a unit test but IIRC files user `src/runtime` are more of a pain to unit test than `src/wcl`